cg/cli/upload.py

```diff
@@ -452,3 +453,39 @@ def validate(context, family_id):
                 click.echo(f"{sample_id}: {mean_coverage:.2f}X - {completeness:.2f}%")
             else:
                 click.echo(f"{sample_id}: sample not found in chanjo", color='yellow')
+
+
+@upload.command()
+@click.option('-c', '--case-id', help='internal case id, leave empty to process all')
+@click.option('-d', '--days-ago', type=int, help='days since solved')
+@click.option('--dry-run', is_flag=True, help='only print cases to be processed')
+@click.pass_context
+def mutacc(context, case_id, days_ago, dry_run):
+
+    """Upload recent cases marked as 'finished' in scout to mutacc"""
+
+    click.echo(click.style('----------------- MUTACC ----------------'))
+
+    scout_api = context.obj['scout_api']
+    mutacc_auto_api = mutacc_auto.MutaccAutoAPI(context.obj)
+
+    mutacc_upload = UploadToMutaccAPI(scout_api=scout_api, mutacc_auto_api=mutacc_auto_api)
+
+    # Get cases to upload into mutacc from scout
+    if days_ago is not None:
+        finished_cases = scout_api.get_solved_cases(days_ago=days_ago)
+    elif case_id is not None:
+        finished_cases = scout_api.get_cases(finished=True, case_id=case_id)
+    else:
+        LOG.info("Please enter option '--case-id' or '--days-ago'")
+
+    for case in finished_cases:
+
+        if dry_run:
+            LOG.info("Would upload case %s to mutacc", case['_id'])
+            continue
+
+        mutacc_upload.extract_reads(case)
```

Comment:
It seems like this method has side effects that we rely on, could we get rid of the side-effects?